lib/path-utils.js

```diff
@@ -0,0 +1,225 @@
+import path from "path"
+import { promises } from "fs"
+const { readFile, stat } = promises
+
+import isPath from "is-valid-path"
+
+import { unique } from "./utils"
+
+/** Converts given path to Posix (replacing \\ with /)
+ * @param {string} givenPath   Path to convert
+ * @returns {string}          Converted filepath
+ */
+export function posixifyPath(givenPath) {
+  return path.normalize(givenPath).replace(/\\/g, "/")
+}
+
+/** Converts given path to Posix (replacing \\ with /) and removing ending slashes
+ * @param {string} givenPath   Path to convert
+ * @returns {string}          Converted filepath
+ */
+export function posixifyPathNormalized(givenPath) {
+  return posixifyPath(givenPath).replace(/[/\\]$/, "")
```

Comment:
path is already posixified
```suggestion
  return posixifyPath(givenPath).replace(/\/$/, "")
```